locales/fr.json

```diff
@@ -33,11 +33,11 @@
   "start.banner.1": "L'Agence du revenu du Canada (ARC) vous a envoyé une lettre concernant le nouveau service",
   "start.banner.2": ". ",
   "start.banner.3": "Vous aurez besoin des renseignements figurant dans cette lettre pour produire votre déclaration de revenus de 2018 à l'aide de ce service.",
-  "start.p1": "Vous pouvez utiliser ce service si l’Ontario était votre province de résidence le 31 décembre 2018. Vos revenus perçus pendant l’année d’imposition 2018 doivent être inférieurs à 35\u00A0000\u00A0$. De plus, vos revenus\u00A0:",
+  "start.p1": "Vous pouvez utiliser ce service si l’Ontario était votre province de résidence le 31 décembre 2018. Vos revenus perçus pendant l’année d’imposition 2018 doivent être inférieurs à 35 000 $. De plus, vos revenus :",
   "start.li1": "doivent provenir d’un emploi, d’un régime de pension, d’un régime enregistré d’épargne-retraite (REER) ou de prestations comme celles d’aide au revenu.",
-  "start.li2": "peuvent comprendre jusqu’à 1\u00A0000\u00A0$ de revenus d’intérêts provenant de sources canadiennes, comme un compte de banque ou d’épargne.",
```

Comment:
Hey Nick, we want to keep these in. Does your IDE automatically take them out?

I just added them the other day: https://github.com/cds-snc/cra-claim-tax-benefits/pull/217

Reply:
I def didn't touch those, it's probably my IDE

Reply:
going to resolve this as we talked about it offline